start.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Does a shell shebang work too? Can't test it myself rn.
```suggestion
#!/bin/sh
```